spec/koans.js

```diff
@@ -192,12 +196,12 @@ describe('The object literal allows for new shorthands. ', () => {
 
   describe('with variables', () => {
     it('the short version for `{y: y}` is {y}', () => {
-      /*.....*/
-      //expect(short).toEqual({y: y});
+      let short = {y: y};
+      expect(short).toEqual({y: y});
```

Comment:
in JavaScript :
const short = {y} 

is actually a shortcut for :
const short = {y: y};